hedera-mirror-rest/api/v1/openapi.yml

```
@@ -2761,24 +2761,53 @@ components:
     NetworkStakeResponse:
       type: object
       required:
+        - max_stake_rewarded
         - max_staking_reward_rate_per_hbar
+        - max_total_reward
         - node_reward_fee_fraction
+        - reserved_staking_rewards
+        - reward_balance_threshold
         - stake_total
         - staking_period
         - staking_period_duration
         - staking_periods_stored
         - staking_reward_fee_fraction
         - staking_reward_rate
         - staking_start_threshold
+        - unreserved_staking_reward_balance
       properties:
+        max_stake_rewarded:
+          description:
```

for multi-line string

```suggestion
          description: |
```

---

Ah, thank you for catching that. Done.